spec/models/conversation_spec.rb

```diff
@@ -5,101 +5,92 @@
 require 'spec_helper'
 
 describe Conversation, :type => :model do
-  before do
-    @user1 = alice
-    @user2 = bob
-    @participant_ids = [@user1.contacts.first.person.id, @user1.person.id]
-
-    @create_hash = {
-      :author => @user1.person,
-      :participant_ids => @participant_ids,
-      :subject => "cool stuff",
-      :messages_attributes => [ {:author => @user1.person, :text => 'hey'} ]
-    }
-  end
+  let(:user1) { alice }
+  let(:user2) { bob }
+  let(:participant_ids) { [user1.contacts.first.person.id, user1.person.id] }
+  let(:create_hash) {{author: user1.person, participant_ids: participant_ids,
+      subject: "cool stuff", messages_attributes: [{author: user1.person, text: "hey"}]}}
+  let(:conversation) { Conversation.create(create_hash) }
+  let(:message_last) { Message.create(author: user2.person, created_at: Time.now + 100,
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.